docs/experiments/join-mode/recursive-joins.md

```diff
@@ -212,6 +212,55 @@ rule:spring-sql-injection: SQLi
 ran 0 rules on 0 files: 1 findings
 ```
 
+## Recursive join with an inline rule example
```

Comment:
This is just a regular inline rule, no recursion here :)

"Inline rule example" is perfect for this

Reply:
Thanks... is this even a correct article for this example than actually? 😅 (Title is Recursive joins)